.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 node_js:
- - "node"
-install:
+ - "lts/*"
```

Comment:
I wonder if we should configure this in the `.nvmrc` file to specify a specific compatible node version?

Reply:
Are you thinking lock it down further with the version LTS is pointing to? (10.15.3)

Reply:
Oh, hmmm our `.nvmrc` is `lts/*`. I know travis reads that when it installs node...curious why we need to do this in the first place 🤔